client/src/modules/list/components/ListHeader.jsx

```diff
@@ -228,19 +250,27 @@ class ListHeader extends PureComponent {
   };
 
   renderName = () => {
-    const { isNameInputVisible, nameInputValue, pendingForName } = this.state;
+    const {
+      isNameInputVisible,
+      isTipVisible,
+      nameInputValue,
+      pendingForName
+    } = this.state;
     const {
       details: { isOwner, name }
     } = this.props;
 
     return isNameInputVisible ? (
-      <NameInput
-        disabled={pendingForName}
-        name={nameInputValue}
-        onClick={isOwner ? this.handleClick : null}
-        onKeyPress={this.handleKeyPress}
-        onNameChange={this.handleNameChange}
-      />
+      <div>
```

Comment:
maybe use <fragment/> here

Reply:
I can't use <Fragment> here. The container of this div is displayed flex with flex-direction: row, so without this div (and with Fragment), the hint appears next to the input field (between the input field and functionality of changing list's type) but not below it.